ci-operator/step-registry/dpf/hypervisor/sanity-existing/dpf-hypervisor-sanity-existing-commands.sh

```
@@ -42,14 +42,37 @@ echo "Remote host: ${REMOTE_HOST}"
 REMOTE_LAST_OPENSHIFT_DPF_DIR_LOCATION="/root/doca8/ci/last-openshift-dpf-dir.sh"
 
 echo "=== DPF Make Target checks on Existing Cluster ==="
 echo "Using openshift-dpf dir from last cluster-deploy: '${REMOTE_LAST_OPENSHIFT_DPF_DIR_LOCATION}'"
 
 ##if ssh ${SSH_OPTS} root@${REMOTE_HOST} "
 
+# update the .env file to shorten the VERIFY_MAX_RETRIES and VERIFY_SLEEP_SECONDS var values
+
+echo "Updating VERIFY_DEPLOYMENT, VERIFY_MAX_RETRIES and VERIFY_SLEEP_SECONDS variable values to true, 4 and 3 respectively in .env file"
+# Using delimiter '|' since we have '/' in the patterns
+if ssh ${SSH_OPTS} root@${REMOTE_HOST} "set -e;\
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)